content/Software Support/Upload/If-Arduino-IDE-detects-a-different-board-than-the-one-youve-connected.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "If Arduino IDE detects a different board than the one you've connected"
+---
+
+Learn what to do if Arduino IDE misidentifies your board.
+
+In this article:
+
+* [Reset manual board selection](#manual-selection)
+* [If your UNO R4 WiFi or Nano ESP32 is identified as a generic ESP32 board](#generic-esp32)
+
+---
+
+<a id="manual-selection"></a>
+
+## Reset manual board selection
+
+While Arduino IDE will detect and select most boards automatically, the default selection can be overridden with a manual selection. This selection will be remembered the next time you connect the board.
+
+To see if a manual selection is being made, look for a **Reset** arrow icon to the right of the board name (under the **Edit** button).
```

Comment:
> Does the arrow show up every time a manual board selection is made?

Only under the following conditions:

- Arduino IDE identified the port as a single specific board name
- The board the user selected is different from the board Arduino IDE identified the port as

For example:

If you connect an UNO R3 board to your computer, Arduino IDE will identify its port as an UNO R3. If you then manually select the Arduino Leonardo for that port, the IDE **will** show the icon.

If you connect a classic Nano to your computer, Arduino IDE will not identify its port (because the classic Nano doesn't have a unique VID/PID). If you  then manually select the Arduino Leonardo for that port, the IDE **will not** show the icon.

Reply:
Thanks for the clarification @per1234